cli/bptest/cmd.go

```diff
@@ -145,3 +146,14 @@ var initCmd = &cobra.Command{
 		return initTest(initTestName)
 	},
 }
+
+var lintCmd = &cobra.Command{
+	Use:   "lint",
+	Short: "Lint metadata.yaml file.",
```

Comment:
We maybe have more lint rules beyond metadata so lets keep generic - "lint blueprint"